.golangci.yml

```diff
@@ -28,8 +27,8 @@ linters:
   - misspell
   - unconvert
   - depguard
-    # To keep this PR small, I'm going to add one linter at a time.
-    #- unused
+  - unused
+    # TODO: enable whitespace linter
```

Comment:
todo should usually follow by an issue number but I guess we will have a followup PR right after this one.

Reply:
yes indeed, it's in fact almost complete!